src/esn.calendar.libs/app/components/event/form/event-form.controller.js

```diff
@@ -288,6 +288,38 @@ function CalEventFormController(
     });
   }
 
+  function _changeOrganizerParticipation(status) {
+    const eventPayload = $scope.event.clone();
+    const currentlyEditedEvent = $scope.editedEvent.clone();
+
+    eventPayload.setOrganizerPartStat(status);
+
+    calEventService.changeParticipation(eventPayload.path, eventPayload, session.user.emails, status, eventPayload.etag)
+      .then(response => {
+        if (!response) {
+          return;
+        }
+
+        calPartstatUpdateNotificationService(status);
+
+        // Re init the event form with the modified event.
+        $scope.event = response;
+        $scope.initFormData();
+
+        // check if the organizer has modified other fields before changing the status
+        if (calEventUtils.hasAnyChange(response, currentlyEditedEvent)) {
+          const changes = currentlyEditedEvent.getChanges(response);
+
+          changes.map(property => {
+            $scope.editedEvent[property] = currentlyEditedEvent[property];
+          });
+        }
+      })
+      .catch(() => {
```

Comment:
An error log could be friendly for DEV here, IMO, by using `$log`

Reply:
done.